backend/internal/api/handler_test.go

```diff
@@ -369,6 +369,66 @@ func TestCreate(t *testing.T) {
 			assert.Equal(t, tt.statusCode, recorder.Code)
 		})
 	}
+
+	t.Run("should return error with status code 406 when request content is corrupted", func(t *testing.T) {
+
+		//given
+		hook := test.NewGlobal()
+		img := model.Image{
+			ID:      fixedID,
+			Content: "notAnImage",
+			Time:    time.Now().Format(time.RFC3339),
```

Comment:
Can't you provide some specific Time object for the testing purposes? Depending on `time.now()` in the unit tests is not considered the best practice.